test/views/CheckYourAnswersViewSpec.scala

```diff
@@ -47,7 +47,7 @@ class CheckYourAnswersViewSpec extends CheckYourAnswersBehaviours with ViewBehav
                  srn: Option[String] = None,
                  hideSaveAndContinueButton: Boolean
                 ): () => HtmlFormat.Appendable = () =>
```

Comment:
Is the ViewSpec for the new checkYourAnswers view missing or merged previously?

Reply:
Added Spec